app/models/sac_cas/event/course.rb

```diff
@@ -217,14 +204,36 @@ def minimum_age
   end
 
   def default_participation_state(participation, for_someone_else = false)
-    return "unconfirmed" if super == "applied" && (places_available? && !automatic_assignment?)
+    if participation.application.blank? || for_someone_else
+      "assigned"
+    elsif places_available? && !automatic_assignment?
+      "unconfirmed"
+    else
+      "applied"
+    end
+  end
 
-    super
+  def total_event_days
+    @total_event_days ||= begin
+      total_event_days = dates.map(&:duration).sum(&:days)
+      total_event_days -= 0.5 if start_point_of_time.to_sym == :evening
```

Comment:
dieselbe Berechnungslogik ist implementiert in `Export::Pdf::Participations::KeyDataSheet::Sections::Table#total_event_days`, sollte diese Methode ev. direkt auf `Event` implementiert werden?